hash/sgi_mips.xml

```diff
@@ -1572,17 +1640,31 @@ license:CC0
 		</part>
 	</software>
 
-	<!-- IndyZone -->
+	<!-- IndiZone -->
 
-	<software name="indizone">
-		<description>IndiZone</description>
+	<software name="indizone_1">
+		<description>IndiZone 1.0</description>
+		<year>1993</year>
+		<publisher>Silicon Graphics</publisher>
+		<info name="version" value="1.0" />
+		<part name="cdrom" interface="cdrom">
+			<feature name="part_number" value="812-8102-001"/>
+			<!-- Origin: private dump -->
+			<diskarea name="cdrom">
+				<disk name="indizone_1" sha1="7dfed1d5b5efb7c773244372d0a24823cf4d5cc4" />
```

Comment:
@darkstar can you double check this dump? it's the only one I can't seem to get IRIX to read at all. `chdman` reports the CD as a `MODE2_RAW`, which is somewhat odd (most SGI CDs I've seen are either `MODE1` or `MODE1_RAW`). If the dump is correct, this is likely a bug, as we currently [hardcode](https://github.com/mamedev/mame/blob/d11915345311eba87c25ee5aec29285badd839af/src/devices/bus/nscsi/cd.cpp#L122) `MODE1` in `nscsi_cd`.

Reply:
Indeed, you're right. I dumped over 20 CDs those last few days, and all with ISObuster. No idea what went wrong there, since that is the only one that was dumped as MODE2_RAW. I re-dumped it and this time it was correctly dumped as MODE1_RAW.
Maybe a bug in ISObuster or something. I will definitely check all dumps from now on to not miss something like that again, thanks for checking (unfortunately my PC is too slow to run IRIX in MAME at any reasonable speed so I couldn't check the CDs in the emulation)

I uploaded the fixed CHD [here](http://files.darkstar.me/indizone.zip), can you try and see if that one works? If it works for you I'll submit another PR with the fixed CHD (alternatively, you can replace the CHD, if you have FTP access)

Reply:
Yup, this one works, thanks for checking! Feel free to send a PR with the fix.

Reply:
okay, will do!

thanks for double-checking!